python-avd/pyavd/_eos_designs/structured_config/underlay/port_channel_interfaces.py

```diff
@@ -72,10 +72,10 @@ def port_channel_interfaces(self: AvdStructuredConfigUnderlay) -> list | None:
 
             if (short_esi := link.get("short_esi")) is not None:
                 port_channel_interface["evpn_ethernet_segment"] = {
-                    "identifier": generate_esi(short_esi, self.shared_utils.evpn_short_esi_prefix),
-                    "route_target": generate_route_target(short_esi),
+                    "identifier": f"{self.shared_utils.evpn_short_esi_prefix}{short_esi}",
+                    "route_target": re.sub(r"(\n{2})(\n{2}):(\n{2})(\n{2}):(\n{2})(\n{2})", r"\1:\2:\3:\4:\5:\6", short_esi),
```

Comment:
Would be better to have this regex used in mutiple places in a util function still

Reply:
ack will do.

Reply:
done